libs/qrencode/Makefile

```diff
@@ -30,7 +30,9 @@ define Package/libqrencode
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=Library for encoding data in a QR Code symbol
+  MENU:=1
```

Comment:
not required

Reply:
It looks really bad without it. Please leave it.